### PR TITLE
feat: CRN-1155 Add API Access to PR Environment in Contentful

### PR DESCRIPTION
**Pull request by peterstarling**
Replaces the current create environment bash script with a TS one and adds the access-key entry bit

### CODE REVIEW

File: .github/workflows/on-push-branch.yml [hunk 1]
```diff
@@ -80,6 +80,7 @@ jobs:
       GP2_SQUIDEX_CI_CLIENT_SECRET: ${{ secrets.GP2_SQUIDEX_CI_CLIENT_SECRET }}
       GP2_SQUIDEX_SHARED_SECRET: ${{ secrets.GP2_SQUIDEX_SHARED_SECRET }}
       CONTENTFUL_MANAGEMENT_TOKEN: ${{ secrets.CONTENTFUL_MANAGEMENT_TOKEN }}
```
Comment (gabiayako) [inline .github/workflows/on-push-branch.yml:82]: just out of curiosity, will this `CONTENTFUL_MANAGEMENT_TOKEN` be shared between CRN and GP2? Just now with your PR I realised that we've not been adding the suffix CRN to most of env vars

**Review by gabiayako — APPROVED** [review]

File: .github/workflows/on-push-branch.yml (hunk 1 above)
Comment (peterstarling) [reply .github/workflows/on-push-branch.yml:82]: actually I think it won't? unlike the access token I think this one works for all spaces. Or am I wrong?

Comment (markcsims) [reply .github/workflows/on-push-branch.yml:82]: I don't think it will be shared?

**Review by markcsims — APPROVED** [review]

**Pull request merged**